### ISSUE TITLE
Brenches disconnected while using wind

### ISSUE THREAD

**Issue by nicks-boson** [joshuazt/JZTREES#4]
Hi, 

(I love this product! great work)

When I use the wind node, the branched are separated from eachother.

<img width="1617" alt="Screenshot 2023-07-10 at 22 36 21" src="https://github.com/joshuazt/JZTREES/assets/113909217/d28bda19-a617-4b5e-9a92-1472aa08ecc6"> WITHOUT WIND NODE
<img width="1633" alt="Screenshot 2023-07-10 at 22 36 27" src="https://github.com/joshuazt/JZTREES/assets/113909217/c3c41d8b-c39c-40f3-869a-f49d3f00c8c0"> WITH WIND NODE

**Comment by joshuazt** [comment]
hey @nicks-boson thanks for getting in contact. This is something it shouldn't be doing. Can you shoot me the .hip file and Ill have a look.

send it to jztrees101@gmail.com

**Comment by joshuazt** [comment]
I just did a little test with the last release and It was behaving as I expected. send me the .hip file when you have a moment :)

**Comment by joshuazt** [comment]
hey @nicks-boson thanks for flagging this, I have just published a fix. the issue was root toggle snapping. I have fixed the bug and also removed the option to use booleans for now as they are broken. 

in the next iteration of jztrees boolean and subdivisional geometry has been completely reworked. little preview images below.

alot of new things are coming, the next verison is pretty much a entire rework of jztrees.


![image](https://github.com/joshuazt/JZTREES/assets/23189306/4e34fe7f-96b3-46a2-b0f0-af9c5d599455)

![image](https://github.com/joshuazt/JZTREES/assets/23189306/167d1741-521a-4a7a-8ad2-f306d5166b2d)
